src/sentry/static/sentry/app/stores/teamStore.jsx

```diff
@@ -51,6 +52,10 @@ const TeamStore = Reflux.createStore({
     this.loadInitialData(this.items.filter(team => team.slug !== slug));
   },
 
+  onCreateSuccess(team) {
+    this.loadInitialData([...this.items, team]);
```

Comment:
why does everything in here use loadInitialData? the extra triggering seems bad

Reply:
It's not an "extra" trigger though? It is only called once.